static/docs/commands-reference/metrics:add.md

```diff
@@ -21,33 +21,30 @@ Alternatively, an output file could be made a metric via `-M` or `-m` parameter
 of the `dvc run` command.
 
 While any text file could be used as a metric file to track, it's recommended to
-use `TSV`, `CSV`, or `JSON` formats. DVC provides a way (see `dvc metrics
-show`), to parse those formats to get to a specific value if file contains
+use `TSV`, `CSV`, or `JSON` formats. DVC provides a way (see `dvc metrics show`), to parse those formats to get to a specific value if file contains
 multiple metrics.
 
 ## Options
 
-* `-t`, `--type` - specify a type of the metric file. Accepted values are:
-`raw`, `json`, `tsv`, `htsv`, `csv`, `hcsv`. Type will be used to determine how
-`dvc metrics show` handles displaying it. This type will be saved into the
-corresponding `.dvc` file and will be used automatically in the `dvc metrics
-show`. `htsv`/`hcsv` are the same `tsv`/`csv` but the values in the first row of
-the file will be used as the field names and should be used to address columns
-in the `--xpath` option. `raw` means that no additional parsing is applied, and
-`--xpath` is ignored. `raw` is the same as default when no type is provided.
-
-* `-x`, `--xpath` - specify a path within a metric file to get a specific metric
-value. Should be used if metric file contains multiple numbers and you need to
-get a only one of them. Only single path is allowed. This path will be saved
-into the corresponding `.dvc` file and will be used automatically in `dvc
-metrics show`. Accepted value depends on the metric file type (`-t` option):
+- `-t`, `--type` - specify a type of the metric file. Accepted values are:
+  `raw`, `json`, `tsv`, `htsv`, `csv`, `hcsv`. Type will be used to determine how
+  `dvc metrics show` handles displaying it. This type will be saved into the
+  corresponding `.dvc` file and will be used automatically in the `dvc metrics show`. `htsv`/`hcsv` are the same `tsv`/`csv` but the values in the first row of
```

Comment:
a bug?

Reply:
@shcheklein seems like it. Let me check on this.